test/api/v01/vrp_test.rb

```diff
@@ -77,6 +77,15 @@ def test_ignore_unknown_parameters
     submit_vrp api_key: 'demo', vrp: vrp
   end
 
+  def test_refute_solver_solver_parameter
```

Comment:
Is repetition of `solver` in purpose or typo?

Reply:
That's a typo, it refutes the association of solver & solver_parameter. But any other ideas for the naming are welcome

Reply:
`solver_and_solver_parameter` maybe?